public/assets/css/style.css

```diff
@@ -726,4 +733,63 @@ button {
         max-width: 30vw;
         margin: auto;
     }
+
+    .game-show-description{
+        width : 25vw;
+        height : unset;
+        margin : unset;
+    }
+
+    .game-show-desription-screen{
+        text-align : center;
+    }
+
+    .unlock-title, .unlock-h2{
+        font-size: 3rem;
+        align-self: center;
+    }
+
+    .agencement-page-show{
+        display : flex;
+        justify-content: space-evenly;
+        height : 100vh;
+    }
+
+    .game-show{
+        align-self: center;
+    }
+
+    .agencement-page-play, .agencement-page-cards-list{
+        display : flex;
+        justify-content: space-around;
+    }
+    
+    .agencement-page-play .unlock-title, .unlock-h2{
+        font-size : 2rem;
+    }
+
+    .agencement-page-play .unlock-h2, .agencement-page-cards-list .unlock-h2{
+        margin-bottom: 3rem;
+    }
+
+    .agencement-page-play div{
+        padding-right: 25rem;
```

Comment:
if you need more than 4 rem for padding and margin, use flexbox or something else